ignite/handlers/checkpoint.py

```diff
@@ -219,6 +220,7 @@ def __init__(
         n_saved: Optional[int] = 1,
         global_step_transform: Callable = None,
         archived: bool = False,
+        filename_pattern: Optional[str] = None,
```

Comment:
We need to update docstring with `filename_pattern`

Reply:
**Args**
```
filename_pattern (str, optional): If ``filename_pattern`` is provided, this pattern will be used to render
checkpoint filenames. If the pattern is not defined, the default pattern would be used.
```
**Note**
```
If only ``global_step_transform`` is defined, then suffix is setup using its return value.
The filename will be `{filename_prefix}_{name}_{global_step}.{ext}`.

If defined ``score_function`` and ``score_name``, then the filename will
be `{filename_prefix}_{name}_{score_name}={score}.{ext}`. If ``global_step_transform`` is provided, then
the filename will be `{filename_prefix}_{name}_{global_step}_{score_name}={score}.{ext}`

If ``filename_pattern`` is not specified, the filename pattern must use the default pattern described above.
If ``filename_pattern`` is given, the filename template would be used to render the filenames.

For example, ``score_name="neg_val_loss"`` and ``score_function`` that returns `-loss` (as objects with highest
scores will be retained), then saved filename will be `{filename_prefix}_{name}_neg_val_loss=-0.1234.pt`. If a 
custom ``filename_pattern`` is given ``filename_pattern="SAVE:{filename_prefix}-{name}-{score_name}-{score}.{ext}`` 
then the saved filename will be `SAVE:{filename_prefix}-{name}-neg_val_loss-0.1234.{ext}`

```
Does this docstring look ok.

Reply:
@Joel-hanson yes, it looks good. Make sure to respect indentation when have multiline text for Args.